src/test/java/net/starlark/java/eval/testdata/int.star

```diff
@@ -7,6 +7,15 @@ assert_eq(5 + 7, 12)
 assert_eq(5 * 7, 35)
 assert_eq(5 - 7, -2)
 
+# negation
+assert_eq(str(-0), "0")
+assert_eq(str(-2), "-2")
+assert_eq(str(--3), "3")
+assert_eq(str(-2147483647), "-2147483647") # -(i32 max value)
```

Comment:
Use the constants defined below, then you won't need the comments.

assert_eq(-maxint, 0 - maxint)
assert_eq(-minint, 0 - minint)
assert_eq(-maxlong, 0 - maxlong)
assert_eq(-minlong, 0 - maxlong)